src/lib/y2partitioner/widgets/btrfs_devices.rb

```diff
@@ -0,0 +1,228 @@
+# encoding: utf-8
+
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "yast2/popup"
+require "cwm"
+require "y2partitioner/widgets/btrfs_metadata_raid_level"
+require "y2partitioner/widgets/btrfs_data_raid_level"
+require "y2partitioner/widgets/btrfs_devices_selector"
+
+module Y2Partitioner
+  module Widgets
+    # Widget grouping a set of widgets related to the selection of Btrfs devices
+    #
+    # It contains widgets to select the metadata/data RAID levels and also to select the devices
+    # used by the filesystem.
+    class BtrfsDevices < CWM::CustomWidget
+      # Constructor
+      #
+      # @param controller [Actions::Controllers::BtrfsDevices]
+      def initialize(controller)
+        textdomain "storage"
+
+        @controller = controller
+      end
+
+      # @macro seeCustomWidget
+      def contents
+        VBox(
+          Left(
+            HVSquash(
+              HBox(
+                metadata_raid_level_widget,
+                HSpacing(1),
+                data_raid_level_widget
```

Comment:
Maybe it's just me, but I find this to be in the wrong order. I would expect:

```
Data RAID Level   -  Metadata RAID Level
```

Instead of the other way around.

Reply:
I would even consider going further, with something like:

```
RAID Level  -  RAID Level for Metadata
```

Reply:
I haven't a strong preference. I would adapt it according to your last suggestion.

Reply:
Then, should we reorder it in the Overview description? Right now, metadata comes first.

Reply:
Yes, it's also counterintuitive there IMO.